rdata/conversion/to_r.py

```diff
@@ -47,25 +45,93 @@ def __call__(self, data: Any, *, encoding: Encoding) -> RObject: # noqa: ANN401
     2: 0x20300,
     3: 0x30500,
 })
+R_MINIMUM_VERSION_WITH_ENCODING: Final[int] = 3
+R_MINIMUM_VERSION_WITH_ALTREP: Final[int] = 3
+
+
+def convert_pd_array_to_np_array(
+        pd_array: Any,  # noqa: ANN401
```

Comment:
I think all Pandas array classes inherit from `ExtensionArray` nowadays.
```suggestion
    pd_array: pd.api.extensions.ExtensionArray,
```

Reply:
Thanks!